draft-ietf-quic-transport.md

```diff
@@ -4554,9 +4554,11 @@ connection IDs gives clients some assurance that the server received the packet
 and that the Version Negotiation packet was not generated by an off-path
 attacker.
 
-As future versions of QUIC may support Connection IDs larger than the version 1
-limit, Version Negotiation packets could carry Connection IDs that are longer
-than 20 bytes.
+Future versions of QUIC may have different requirements for the lengths of
+connection IDs larger than version 1. In particular, connection IDs might not
```

Comment:
```suggestion
connection IDs. Other versions might allow connection IDs to be longer.  Other
versions might permit shorter or longer connection IDs to be included in packets
sent by a client to open a connection. A server MUST send a Version Negotiation
packet in response to a packet containing a connection ID smaller than 8 bytes
or larger than 20 bytes.
```

Reply:
@martinthomson Isn't the second sentence (i.e., "Other versions might allow connection IDs to be longer.") excessive? The next sentence covers that, I think.